nvflare/private/fed/server/server_commands.py

```diff
@@ -81,7 +88,12 @@ def process(self, data: Shareable, fl_ctx: FLContext):
         if server_runner:
             server_runner.abort(fl_ctx)
             # wait for the runner process gracefully abort the run.
-            time.sleep(3.0)
+            engine = fl_ctx.get_engine()
+            start_time = time.time()
+            while engine.engine_info.status != MachineStatus.STOPPED:
+                time.sleep(1.0)
+                if time.time() - start_time > 30.0:
+                    break
```

Comment:
If this period is longer than 30 seconds, does this mean something is wrong?

Should we return a warning message?

Reply:
abort_job command only needs to respond the command been delivered to the child process. No need to send back warning message.